integration/setup_test.go

```diff
@@ -31,22 +31,24 @@ func (s *integrationTestSuite) Setup() {
 		o := obj
 		// check if object already exists
 		var existingObj client.Object
-		_ = integration.Client.Get(ctx, client.ObjectKey{
+		err := integration.Client.Get(ctx, client.ObjectKey{
 			Namespace: o.GetNamespace(),
 			Name:      o.GetName(),
 		}, existingObj)
 
-		if existingObj == nil {
+		if err != nil {
```

Comment:
```suggestion
		if errors.IsNotFound(err) {
```
To provide distinction between object was just not found v/s some weird error happened while GETing that object.